Makefile

```diff
@@ -1,24 +1,24 @@
 # Makefile
 #
 build:
-	cp _tags.full _tags
-	ocaml pkg/pkg.ml build 
+	@echo "src-protocol-exts-pp-vg\nsrc-protocol-exts-vg" > jbuild-ignore 
```

Comment:
One small suggestion is to add the generated `jbuild-ignore` to `.gitignore`. I know the makefile removes it again but e.g. if the build fails it's not removed and might be committed accidentally.